ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/Toggle.kt

```diff
@@ -66,11 +66,6 @@ fun Toggle(
         Box(
             Modifier.size(ToggleWidth, ToggleHeight)
                 .clip(ToggleBackgroundShape)
-                .clickable(
-                    onClick = { onToggle(!enabled) },
-                    interactionSource = interactionSource,
-                    indication = LocalIndication.current,
-                )
```

Comment:
These were just redundant, but not incorrect, right? Having them set here gives the toggle on its own a ripple when clicked (directly) that's why they were used.

Reply:
This is actually something I was thinking a lot. From assistive tech-navigation-POV, they do add extra tab stops - I'm wondering, is there a way to make a component non-focusable but clickable, because that would be the best option here 🤔

Reply:
I guess the weird part is that we don't use this directly (so far), only embedded into `SettingsItem`, where we clear its semantics so that we can put the listener on the larger piece of UI (the whole row) instead and not just the toggle itself.

But in theory this toggle could be used on its own as well somewhere else.